README.md

```diff
@@ -105,21 +175,23 @@ Launch camera or play bag file
 * Plug in a rgb-d camera or download and launch the example .bag file.
 
 ```
+
 # with kinect: plug the camera into your computer
 roslaunch openni2_launch openni2.launch depth_registration:=true
 # set camera resolution to QVGA
 rosrun dynamic_reconfigure dynparam set /camera/driver ir_mode 7
 rosrun dynamic_reconfigure dynparam set /camera/driver color_mode 7
 rosrun dynamic_reconfigure dynparam set /camera/driver depth_mode 7
 
-
 # with .bag file: 
 # example of ifco bag https://tubcloud.tu-berlin.de/s/yKQrraTdSsb54TC
 # example of table top scenario (link follows)
 # use ros sim time
 rosparam set use_sim_time true
 roslaunch openni2_launch openni2.launch depth_registration:=false
-rosbag play -l Ifco_vision_test.bag  (or any other bag)
+rosbag play -l ifco_example.bag  (or any other bag)
+# visualize in rviz:
+rosrun rviz rviz -d `rospack find ecto_rbo_yaml`/cfg/ifco_example.rviz
 ```
 
 Execute vision
```

Comment:
The Execution lacks some instructions: where is the bag, how to lunch it, which topics to publish, and what is the expected outcome (e.g., a picture of RVIZ visualisation). 
The simple 'demo_vision.yaml` should be also added.